content/api/commands/attachfile.md

```diff
@@ -0,0 +1,231 @@
+---
+title: attachFile
+---
+
+Attaches a file or files to an HTML5 input element or simulates dragging a file or files into the browser.
+
+## Syntax
+
+```javascript
+.attachFile(file)
+.attachFile(file, options)
+.attachFile([file], options)
+.attachFile([file], options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
```

Comment:
We can get a little more of this section in view by removing the extra couple of spaces in front of this fenced in block.

```suggestion
```javascript
cy.get('input[type=file]').attachFile('file.json')
cy.get('input[type=file]').attachFile(['file.json', 'file2.json'])
cy.get('input[type=file]').attachFile({
  contents: Buffer.from('foo'),
  fileName: 'file.json',
  lastModified: Date.now(),
})
```

Reply:
This one may have gotten wonky via GH suggestions with the multiple sets of ```. The idea is just to close the extra white space.

Reply:
Having the spaces looks nicer in the markdown, but markdown isn't the actual target format. :)

Done.